src/gql/generated/types.ts

```diff
@@ -626,7 +626,14 @@ export type UserPatchesQueryVariables = {
 };
 
 
-export type UserPatchesQuery = { userPatches: { patches: Array<{ projectID: string, description: string, status: string, createTime?: Maybe<Date>, builds: Array<{ id: string, buildVariant: string, status: string, predictedMakespan: number, actualMakespan: number }> }> } };
+export type UserPatchesQuery = { userPatches: { filteredPatchCount: number, patches: Array<{ id: string, projectID: string, description: string, status: string, createTime?: Maybe<Date>, builds: Array<{ buildVariant: string, status: string }> }> } };
+
+export type PatchBuildVariantsAndStatusQueryVariables = {
+  id: Scalars['String'];
```

Comment:
I noticed this before its weird how its generating single quotes for you but double quotes for me. Is prettier running on this file?

Reply:
`npm run codegen` is outputting single quotes for me

Reply:
saving with prettier will generate double quotes though

Reply:
Thats strange ill try to figure out whats going on. So its more consistent from now on

Reply:
What if we set up some precommit hooks to run prettier and eslint on our code before we commit or push up changes? @tgrander  @SupaJoon  what do you think?

Reply:
that would be great